docs/undergrad.yaml

```diff
@@ -58,7 +58,7 @@ Linear algebra:
   Structure theory of endomorphisms:
     eigenvalue: 'module.End.has_eigenvalue'
     eigenvector: 'module.End.has_eigenvector'
-    diagonalization: 'https://en.wikipedia.org/wiki/Diagonalizable_matrix'
+    diagonalization: 'is_self_adjoint.diagonalization'
```

Comment:
Not yet -- `is_self_adjoint.diagonalization` is already linked below in the section on bilinear forms; this should rather be the _definition_ of a diagonalization of a matrix.